sdk/lib/convert/json.dart

```diff
@@ -185,6 +226,43 @@ class JsonCodec extends Codec<Object?, String> {
 }
 
 /// This class converts JSON objects to strings.
+///
+/// Example:
+///
+/// ```dart
+/// const JsonEncoder encoder = JsonEncoder();
+/// var values = {};
+/// values['text'] = 'foo';
+/// values['value'] = 2;
+///
+/// var jsonData = {};
+/// jsonData['content'] = values;
```

Comment:
Not sure this extra object adds anything to the example?

Reply:
keeping it simple by removing that. if needed something similar can be added later.